input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md

```diff
@@ -0,0 +1,8 @@
+### Usage scenarios
+The following are supported usage scenarios for this profile:
+
+- Query for a request for a diagnostic service for a patient
+- Record or update a request for a diagnostic service for a patient
+
+### Profile specific implementation guidance
+- The use of coding can vary significantly across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.
```

Comment:
Add profile-specific implementation guidance that this profile provides a shared base that is common across requests for specific domains and where a profile is defined for a specific domain that profile SHALL be used. Then sub-bullet points path requests SHALL use the path profile, and the imaging requests SHALL use the imaging profile.